### PR TITLE
changing the logo

### PR DESCRIPTION
**Pull request by sd33m**

### CODE REVIEW

**Comment by qdozaq** [comment]
I'm seeing this when I look at this branches readme 
image

What is this change supposed to be for?

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <p align="center">
-    <img src="https://github.com/factn/resilience-app/blob/master/src/img/logo.png?raw=true" />
+    <img src="https://lh6.googleusercontent.com/hB9eX1x2uNM2Ozqb7aVGb72a95eVdy_dCT6TS_ImL-aqg4SHbEMu6MNxWv5-Nzhjn0jRP3DvbmDbBXL5xT5M=w1920-h914" />
```
Comment (mat10tng) [inline README.md:2]: image should upload using github upload system

**Comment by mat10tng** [comment]
Can we have a closure on this, please?

**Comment by jwu910** [comment]
I moved this to review, but i realize we don't have an issue to attach it to yet

**Pull request closed without merging**